examples/kubeflow/README.md

```diff
@@ -0,0 +1,255 @@
+
+This example can also be run from the `notebook.ipynb` included in this directory.
+
+## Prerequisites
+
+This guide assume that Kubeflow is already installed in Kubernetes cluster. See [Kubeflow Manifests](https://github.com/kubeflow/manifests) for installation instructions.
+
+Install BentoML cloud native components and custom resource definitions.
+
+```bash
+kustomize build bentoml-yatai-stack/default | kubectl apply -n kubeflow --server-side -f -
+```
+
+Install the required packages to run this example.
+
+```bash
+pip install -r requirements.txt
+```
+
+## Download Kaggle Dataset
+
+Set Kaggle username and key as environment variables. Accepting the [rules of the competition](https://www.kaggle.com/competitions/ieee-fraud-detection/rules) is required for downloading the dataset.
+
+```bash
+export KAGGLE_USERNAME=
+export KAGGLE_KEY=
+```
+
+Download Kaggle dataset.
+
+```bash
+kaggle competitions download -c ieee-fraud-detection
+rm -rf ./data/
+unzip -d ./data/ ieee-fraud-detection.zip && rm ieee-fraud-detection.zip
+```
+
+## Train Models
+
+In this demonstration, we'll train three fraud detection models using the Kaggle IEEE-CIS Fraud Detection dataset. To showcase saving and serving multiple models with Kubeflow and BentoML, we'll split the dataset into three equal-sized chunks and use each chunk to train a separate model. While this approach has no practical benefits, it will help illustrate how to save and serve multiple models with Kubeflow and BentoML.
+
+```python
+import pandas as pd
+
+df_transactions = pd.read_csv("./data/train_transaction.csv")
+
+X = df_transactions.drop(columns=["isFraud"])
+y = df_transactions.isFraud
+```
+
+Define the preprocessor.
+
+```python
+from sklearn.impute import SimpleImputer
+from sklearn.compose import ColumnTransformer
+from sklearn.impute import SimpleImputer
+from sklearn.preprocessing import OrdinalEncoder
+
+numeric_features = df_transactions.select_dtypes(include="float64").columns
+categorical_features = df_transactions.select_dtypes(include="object").columns
+
+preprocessor = ColumnTransformer(
+    transformers=[
+        ("num", SimpleImputer(strategy="median"), numeric_features),
+        (
+            "cat",
+            OrdinalEncoder(handle_unknown="use_encoded_value", unknown_value=-1),
+            categorical_features,
+        ),
+    ],
+    verbose_feature_names_out=False,
+    remainder="passthrough",
+)
+
+X = preprocessor.fit_transform(X)
+```
+
+Define our training function with the number of boosting rounds and maximum depths.
+
+```python
+import xgboost as xgb
+
+def train(n_estimators, max_depth):
+    return xgb.XGBClassifier(
+        tree_method="hist",
+        n_estimators=n_estimators,
+        max_depth=max_depth,
+        eval_metric="aucpr",
+        objective="binary:logistic",
+        enable_categorical=True,
+    ).fit(X_train, y_train, eval_set=[(X_test, y_test)])
```

Comment:
do you mean to add X_train, y_train, X_test, y_test here as parameters of `train` function?